{
    "checksum": "90053cbb6fb05ba1a70c61c84cee9f3fc43b0cdb0f85962d99d6aebbf5549f27",
    "file": "exception.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 4,
    "package": "backblaze-b2",
    "path": "backblaze-b2/1.3.8-5/b2/sync/exception.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/backblaze-b2/1.3.8-5/",
        "metric": {
            "size": 1288
        },
        "pts_link": "https://tracker.debian.org/pkg/backblaze-b2",
        "sloc": [
            [
                "python",
                10230
            ],
            [
                "sh",
                149
            ],
            [
                "makefile",
                40
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/python-team/packages/backblaze-b2",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/b/backblaze-b2/1.3.8-5/b2/sync/exception.py",
    "stat": {
        "perms": "rw-r--r--",
        "size": 856,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.3.8-5"
}